### DISCUSSION TITLE
Desync Kick

### DISCUSSION THREAD

**Discussion by XL-Faith** [YimMenu/YimMenu#2508] [Q&A]
Everytime I inject the menu, I can't get in any public session without getting desync kicked, I end up always in a solo public session and the only way is by getting the original files from the game reinstalled. Anyone had this problem that knows how to solve it?

**Comment by Josephyess** [comment]
just use VPN

**Reply by XL-Faith** [reply]
It was solved with the new nightly release

**Reply by Josephyess** [reply]
good 4u
